src-common/main/scala/uk/gov/hmrc/audit/handler/HttpHandler.scala

```diff
@@ -62,13 +62,11 @@ abstract class HttpHandler(
             HttpResult.Malformed
           }
         }.recover {
-          case e: TimeoutException =>
-            HttpResult.Failure("Error opening connection, or request timed out", Some(e))
-          case e: IOException =>
-            HttpResult.Failure("Error opening connection, or request timed out", Some(e))
+          case e: Throwable =>
+            HttpResult.Failure("Error opening connection or sending request", Some(e))
         }
     } catch {
-      case t: Throwable =>
-        Future.successful(HttpResult.Failure("Error sending HTTP request", Some(t)))
+      case e: Throwable =>
+        Future.successful(HttpResult.Failure("Error opening connection or sending request", Some(e)))
```

Comment:
would we ever want to differentiate between the 2 failures above? They share the same message but are in different parts of the code

Reply:
I was very on the fence about this.  I might put in something token (like '(async)' at the end of the future one and '(sync)' at the end of the direct throw.